gcp/tests/conftest.py

```
@@ -15,6 +15,17 @@
 import sys
 from pathlib import Path
 
+
+def pytest_configure(config):
```

Ruff (0.15.14)

[warning] Unused function argument: `config`

(ARG001)